lib/aliquot/payment.rb

```diff
@@ -133,15 +133,15 @@ def check_signature
             key.verify(new_digest, message_signature, signed_string_message)
           end.any?
 
-        raise InvalidSignatureError unless success
+        raise InvalidSignatureError, 'signature of signedMessage did not match' unless success
```

Comment:
Why suddenly the past tense?

Reply:
Not for any particular reason.

The reason I added an error message is simply to be more concrete about the signature of `signedMessage` being wrong.

The past tense is because I read it as something like
> The _signature of signedMessage did not match_ when we checked it.

I guess in present tense it would be
> The _signature of signedMessage does not match_.


It should be in present tense.
Good catch :+1:

Reply:
hehe, I am partly sorry about the way I formulated it. Perhaps better: For the tense in the error messages to be consistent, I think this message should be in present tense (and the same for lines 144 and 155; and of course specs might need adjustments).